### PR TITLE
Scratch 3.0 Updated version

### PR DESCRIPTION
**Pull request by apple502j**
* Changes "Tips" to "Ideas"
* Translation updates:
  - [x] German (de.json)
  - [x] English (en.json)
  - [x] French (fr.json)
  - [x] Indonesian (id.json)
  - [x] Japanese (ja.json)
  - [x] Dutch (nl.json)
  - [x] Documentation (qqq.json)
  - [x] Russian (ru.json) (Done in #49)
  - [x] Chinese (zh-hans.json and zh-hant.json)

Should be [blocked] until Jan 2nd.

### CODE REVIEW

File: i18n/id.json [hunk 1]
```diff
@@ -44,13 +44,13 @@
...
 	"scratchwikiskin-footer-help": "Bantuan Untuk Penyunting",
-	"scratchwikiskin-footer-tips": "Tips",
+	"scratchwikiskin-footer-ideas": "Tips",
 	"scratchwikiskin-footer-faq": "Pertanyaan yang Sering Diajukan",
 	"scratchwikiskin-footer-offline": "Editor Luring",
 	"scratchwikiskin-footer-store": "Toko Scratch",
 	"scratchwikiskin-footer-donate": "Donasi",
 	"scratchwikiskin-footer-legal-title": "Legal",
-	"scratchwikiskin-footer-privacy": "Kebijakan Privasi",
+	"scratchwikiskin-footer-privacy": "Kebijakan Privsasi",
```
[the remark this answers is not in the document]
Comment (apple502j) [reply i18n/id.json]: I made a huge mistake, as huge as Mt.Fuji...

**Comment by Kenny2github** [comment]
(thank you for understanding what I meant when I specified the wrong strings for the wrong languages!)

**Comment by apple502j** [comment]
Changed credits url as well (source: LLK/scratch-www#2524

**Comment by rumanti** [comment]
Oops I'm not sure if I did the right thing

**Comment by Kenny2github** [comment]
@Rumanti that's good, thanks.

**Comment by Kenny2github** [comment]
Fixed the German translations using the website strings.

**Pull request merged**